compiler/rustc_parse/src/parser/stmt.rs

```diff
@@ -55,6 +55,19 @@ impl<'a> Parser<'a> {
             return Ok(Some(stmt.into_inner()));
         }
 
+        if self.token.is_keyword(kw::Mut) && self.is_keyword_ahead(1, &[kw::Let]) {
+            self.bump();
+            let mut_let_span = lo.to(self.token.span);
+            self.struct_span_err(mut_let_span, "invalid variable declaration")
+                .span_suggestion(
+                    mut_let_span,
+                    "move the `mut` to behind the `let`",
```

Comment:
Wouldn't "move the `mut` after the `let`" sound better?

Reply:
which message looks better?

Reply:
I think that the current "move the ```mut``` to behind the ```let```" is more clear, since  "move the ```mut``` after the ```let```" may mean to move the ```let``` first then move the ```mut``` (extremely slight ambiguity). But, the current phrasing can be improved but I can't think of a better one currently.

Reply:
An alternative to my proposal to address that potential ambiguity could be to use "move the `mut` right after the `let`".

For a different phrasing: "invert the order of `mut` and `let`".

Reply:
The different phrasing sound good, but I would prefer using "switch" instead of "invert".